clang/test/Sema/aarch64-sme-func-attrs.c

```diff
@@ -483,14 +483,16 @@ void just_fine(void) {}
 
 __arm_locally_streaming
 __attribute__((target_version("sme2")))
-void just_fine_locally_streaming(void) {}
+void incompatible_locally_streaming(void) {}
+// expected-error@-1 {{attribute 'target_version' multiversioning cannot be combined with attribute '__arm_locally_streaming'}}
+// expected-cpp-error@-2 {{attribute 'target_version' multiversioning cannot be combined with attribute '__arm_locally_streaming'}}
```

Comment:
I missed this before... why?

cc @sdesmalen-arm

Reply:
I believe this has to do with the order of parsing. Before only after we saw the default we then acknowledged the sme2 version, so this diagnostic was missed. But I am not 100% sure this is the reason.

Reply:
I think this is a regression, but it's fine to fix it in another patch. The locally streaming function has the same ABI as the non-streaming one, so FMV _should_ be possible between them (unlike w/ a streaming one). LIkewise, it should be possible to FMV between streaming compatible functions and normal ones.